src/main/osd/osd_elements.c

```diff
@@ -1858,6 +1863,8 @@ void osdElementsInit(bool backgroundLayerFlag)
 {
     backgroundLayerSupported = backgroundLayerFlag;
     activeOsdElementCount = 0;
+    pt1FilterInit(&batteryEfficiencyFilt, 0.0f);
```

Comment:
```suggestion
    pt1FilterInit(&batteryEfficiencyFilt, pt1FilterGain(dT, EFFICIENCY_CUTOFF_HZ));
```
You need to replace `dT` with the looptime of the batt efficiency calculation.